packages/react-native-skia/components/textinput/TextinputAutoFocus.js

```diff
@@ -0,0 +1,66 @@
+import React, {useRef} from "react";
```

Comment:
than adding new test app , can autoFocus be used in the existing test app itself.
This will help to identify regressions if any.

Reply:
ya, sure.